src/xtgeo/xyz/polygons.py

```diff
@@ -153,22 +156,29 @@ def polygons_from_roxar(
         stype: RMS folder type, 'horizons' (default), 'zones', 'clipboard',
             'faults', 'general2d_data'
         realisation: Realisation number, default is 0
+        attributes: Polygons can store an attrubute (e.g. a fault name) per polygon,
+            i.e. per "POLY_ID")
 
     Example::
 
         import xtgeo
         mysurf = xtgeo.polygons_from_roxar(project, 'TopAare', 'DepthPolys')
 
     .. versionadded:: 2.19 general2d_data support is added
+    .. versionadded:: 3.x support for polygon attributes (other than POLY_ID)
     """
 
+    stype = "horizons" if stype is None else stype
```

Comment:
Sometimes my eyes prefer this 🙂 .. not sure why it is needed here though? wouldn't the default value in the function signature be enough? I see the typehint is optional, but will it ever come in as None? 
```suggestion
    stype = stype or "horizons"
```

Reply:
There are some tests setting horizons="" which should fail (i.e. "" is not None) so will probably keep as is.